build_tools/stable_diffusion_testing.py

```diff
@@ -54,6 +54,8 @@ def test_loop(device="vulkan", beta=False, extra_flags=[]):
         extra_flags.append("--beta_models=True")
     for import_opt in import_options:
         for model_name in hf_model_names:
+            if "Linaqruf" in model_name:
+                continue
```

Comment:
maybe good to qualify with the model and not everything from Linaqruf/ but it is ok to land for now.